### ISSUE TITLE
Why Babel is listed as a dependency?

### ISSUE THREAD

**Issue by flowck** [levibostian/ExpressjsBlanky#2]
@levibostian impressive work. In the node community, I believe we don't have a heavy used pattern for backend development or api development, I've seen many applications written in nodejs with different approach when talking about file organization, patterns or way to do x/y. 

Since the latest versions of nodejs have a great support for ES6/7, I am wondering why did you adopted Babeljs. 

Thanks.

**Comment by levibostian** [comment]
Great question. 

TL;DR So, I am just used to it and find it a nice tool. That's the main reasons. 

First reason is because I have very recently learned about ES6/ES7 support in Nodejs. When I was first introduced to Babel, I thought you *had* to use it for your project to use any of the ES6/ES7 code. I simply didn't know you don't have to. 

Second reason is that it's easier. I find it easier to simply use Babel for all of my ES6/ES7 code I want to write because it just works rather then looking up Nodejs support for a certain language feature. For example: I have some ES6 classes with static properties. Nodejs does not support this feature but of course Babel does. It's just easier knowing that babel can handle it to do less research.

**Comment by levibostian** [comment]
Also, thanks for the kind comments about my "impressive work" 😄. I appreciate that. 

I am not a pro at my Nodejs skills, but I am pretty proud of what I have built here. It's a fun setup to work with.

**Comment by flowck** [comment]
@levibostian It makes sense.
